apps/web/src/app/[locale]/signups/[signupId]/[signupEditToken]/page.tsx

```diff
@@ -0,0 +1,98 @@
+import { notFound } from "next/navigation";
+import { getSignup } from "../../../../../lib/api/external/ilmomasiina";
+import { openGraphImage } from "../../../../shared-metadata";
+import {
+  deleteSignUpAction,
+  saveSignUpAction,
+} from "../../../../../lib/api/external/ilmomasiina/actions";
+import { getScopedI18n } from "../../../../../locales/server";
+import { SignupForm } from "./signup-form";
+
+interface PageProps {
+  params: {
+    signupId: string;
+    signupEditToken: string;
+  };
+}
+
+export const generateMetadata = async ({
+  params: { signupId, signupEditToken },
+}: PageProps) => {
+  const signupInfo = await getSignup(signupId, signupEditToken);
+  const t = await getScopedI18n("ilmomasiina.form");
+
+  if (!signupInfo.ok) {
+    return {};
+  }
+
+  return {
+    title: `${t("Edit sign up")} - ${signupInfo.data.event.title}`,
+    description: signupInfo.data.event.description,
+    openGraph: {
+      ...openGraphImage,
+    },
+  };
+};
+
+export default async function Page({
+  params: { signupId, signupEditToken },
+}: PageProps) {
+  const signupInfo = await getSignup(signupId, signupEditToken);
+
+  if (!signupInfo.ok && signupInfo.error === "ilmomasiina-signup-not-found") {
+    notFound();
+  }
+
+  if (!signupInfo.ok) {
+    throw new Error("Failed to fetch signup info");
+  }
+
+  const t = await getScopedI18n("ilmomasiina.form");
+
+  return (
+    <main
+      id="main"
+      className="relative mb-8 flex flex-col items-center gap-2 md:gap-6"
+    >
+      <div className="relative flex max-w-4xl flex-col items-center gap-8 p-4 md:p-6">
+        <hgroup className="space-y-4 text-pretty">
+          <h1 className="font-mono text-2xl md:text-4xl">
+            {t("Edit sign up")} - {signupInfo.data.event.title}
+          </h1>
+          <p>
+            {signupInfo.data.signup.status === "in-queue"
+              ? t("You are in queue at position {position}", {
+                  position: signupInfo.data.signup.position,
+                })
+              : t(
+                  "You are in the quota {quotaName} at position {position}/{quotaSize}",
+                  {
+                    quotaName: signupInfo.data.signup.quota.title,
+                    position: signupInfo.data.signup.position,
+                    quotaSize: signupInfo.data.signup.quota.size,
+                  },
+                )}
+          </p>
+        </hgroup>
+        <SignupForm
+          signupId={signupId}
+          signupEditToken={signupEditToken}
+          signup={signupInfo.data.signup}
+          event={signupInfo.data.event}
+          saveAction={saveSignUpAction}
+          deleteAction={deleteSignUpAction}
+        />
+        <footer className="text-sm text-gray-800">
+          <p>
+            {t(
+              "Note: Sign up integration is in beta, if you encounter any issues you can sign up directly on the event page: {eventUrl}",
```

Comment:
This currently makes the link possibly wrap across lines and not act as an actual link, i.e. `anchor.`
How about the following diff (can be applied with `git apply -`):
```shell
diff --git a/apps/web/src/app/[locale]/signups/[signupId]/[signupEditToken]/page.tsx b/apps/web/src/app/[locale]/signups/[signupId]/[signupEditToken]/page.tsx
index d07b79e..cbc2560 100644
--- a/apps/web/src/app/[locale]/signups/[signupId]/[signupEditToken]/page.tsx
+++ b/apps/web/src/app/[locale]/signups/[signupId]/[signupEditToken]/page.tsx
@@ -85,11 +85,16 @@ export default async function Page({
         <footer className="text-sm text-gray-800">
           <p>
             {t(
-              "Note: Sign up integration is in beta, if you encounter any issues you can sign up directly on the event page: {eventUrl}",
-              {
-                eventUrl: `${process.env.PUBLIC_ILMOMASIINA_URL ?? ""}/signup/${signupId}/${signupEditToken}`,
-              },
-            )}
+              "Note: Sign up integration is in beta, if you encounter any issues you can sign up directly on the event page",
+            )}{" "}
+            <a
+              className="font-semibold underline"
+              href={`${process.env.PUBLIC_ILMOMASIINA_URL ?? ""}/signup/${signupId}/${signupEditToken}`}
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              {t("here")}
+            </a>
           </p>
         </footer>
       </div>
diff --git a/apps/web/src/locales/en.ts b/apps/web/src/locales/en.ts
index 250aa03..a4a2598 100644
--- a/apps/web/src/locales/en.ts
+++ b/apps/web/src/locales/en.ts
@@ -47,8 +47,9 @@ const en = {
   "ilmomasiina.form.optional": "optional",
   "ilmomasiina.form.Shown in the public list of sign ups":
     "Shown in the public list of sign ups",
-  "ilmomasiina.form.Note: Sign up integration is in beta, if you encounter any issues you can sign up directly on the event page: {eventUrl}":
-    "Note: Sign up integration is in beta, if you encounter any issues you can sign up directly on the event page: {eventUrl}",
+  "ilmomasiina.form.Note: Sign up integration is in beta, if you encounter any issues you can sign up directly on the event page":
+    "Note: Sign up integration is in beta, if you encounter any issues you can sign up directly on the event page",
+  "ilmomasiina.form.here": "here",
   "ilmomasiina.form.First name": "First name",
   "ilmomasiina.form.Last name": "Last name",
   "ilmomasiina.form.Email": "Email",
diff --git a/apps/web/src/locales/fi.ts b/apps/web/src/locales/fi.ts
index 6be7f39..365f54f 100644
--- a/apps/web/src/locales/fi.ts
+++ b/apps/web/src/locales/fi.ts
@@ -47,8 +47,9 @@ const fi = {
   "ilmomasiina.form.optional": "valinnainen",
   "ilmomasiina.form.Shown in the public list of sign ups":
     "Näytetään julkisessa osallistujalistassa",
-  "ilmomasiina.form.Note: Sign up integration is in beta, if you encounter any issues you can sign up directly on the event page: {eventUrl}":
-    "Huomio: Ilmoittautuminen suoraan sivuilla on beta-vaiheessa, jos kohtaat ongelmia voit ilmoittautua tapahtuman sivulla ilmomasiinassa: {eventUrl}",
+  "ilmomasiina.form.Note: Sign up integration is in beta, if you encounter any issues you can sign up directly on the event page":
+    "Huomio: Ilmoittautuminen suoraan sivuilla on beta-vaiheessa, jos kohtaat ongelmia voit ilmoittautua tapahtuman sivulla ilmomasiinassa",
+  "ilmomasiina.form.here": "täällä",
   "ilmomasiina.form.First name": "Etunimi",
   "ilmomasiina.form.Last name": "Sukunimi",
   "ilmomasiina.form.Email": "Sähköposti",
```
Not the prettiest and finnish text is a bit clunky, but you get the gist 😄

Reply:
Also, I wonder if we could also link to the "Give feedback" here somehwere as well?

Reply:
There might be better support for links (or HTML/JSX) in the i18n lib, would need to check. I don't think this is too great since the link text "here" is not good.

This was more meant as a temporary thing for maybe 1-2 sign up at most, that's why I didn't bother too much with making it a proper link since it works 😅

Reply:
Won't fix, will just remove the notice after is validated working on Monday (Nettisivusitsit signup)